app/src/app/components/contents/packets/PacketDetails.tsx

```diff
@@ -43,7 +48,7 @@ export default function PacketDetails() {
                 {hasParameters() && <ParameterList parameters={packet.parameters}/>}
             </div>
             <div data-testid="runner-content" className="content-box">
-                <p>content</p>
+                {hasHash() && <PacketFile hash={packet.files[0].hash}/>}
```

Comment:
I don't think we can assume that the first file in a packet will be a displayable html file, can we? Orderly has the concept of a 'primary resource' or something like that, but have we decided what we're going to do for Packit? Just display the first html file we find in the list, if any?

Reply:
Agreed, I think I mentioned this will be fixed in future ticket when we actually return html in the demo files.

Reply:
The logic in orderlyweb is: 
* if the first file is of a displayable type, then display it
* otherwise do not display anything